lua/supermaven-nvim/commands.lua

```diff
@@ -43,6 +43,22 @@ M.setup = function()
   vim.api.nvim_create_user_command("SupermavenClearLog", function()
     api.clear_log()
   end, {})
+
+  vim.api.nvim_create_user_command("SupermavenDisable", function()
+    api.stop()
+    vim.cmd([[
+      let g:SUPERMAVEN_DISABLED = 1
+    ]])
+    vim.notify("Supermaven is disabled globally.", vim.log.levels.WARN)
```

Comment:
If you can use the logger like `log:info` would be better for the people who have `log_level = "off"` in their configs.

Also it keeps the format of the rest of notifies that we have right now `[supermaven-nvim] %s` and passing the options of `{ title = "Supermaven" }` useful to keep consistency for the people using a notification plugin like [nvim-notify](https://github.com/rcarriga/nvim-notify) for example.

Reply:
Alright thanks I will do that